packages/gatsby-plugin-utils/src/__tests__/is-feature-available.ts

```diff
@@ -0,0 +1,24 @@
+import { hasFeature } from "../is-feature-available"
+import apis from "gatsby/apis.json"
+
+jest.mock(`gatsby/apis.json`, () => {
+  return {}
+})
+
+describe(`hasFeature`, () => {
+  it(`should return true if gatsby has the feature`, () => {
```

Comment:
Let's add a test case for when people have a typo in their argument of `hasFeature` but the feature exists. Then in the future we could add a "did you mean?" and test that

Reply:
I'm confused, should I add did-you-mean now? Cause else how would I be able to write a test case? :p

Reply:
Sorry for the confusion 😅  I meant:

- Add a test case for a typo now => Should resolve to "false"
- If we ever add a "did you mean" in the future, we could adjust that test to also get that message back